### ISSUE TITLE
Recalling message by replying /rm doesn't work

### ISSUE THREAD

**Issue by catbaron0** [blueset/efb-telegram-master#89]
We can't recall a message from remote chat by replying `/rm` to it anymore.
Edit the message to `rm\`` still works, though.
I think it is related to the update that reduce the size of database.

I temporarily  fixed it by editing `master_message.py:493` from
```
if not msg_log or msg_log.slave_member_uid == self.db.FAIL_FLAG or not msg_log.pickle:
```
to
```
if not msg_log or msg_log.slave_member_uid == self.db.FAIL_FLAG:
``` 
but i'm not sure if this fix could cause any other bugs.

**Comment by blueset** [comment]
Thanks for bringing this up, will fix this in the next release.

**Comment by blueset** [comment]
Fixed in 2.0.0b37.
